apps/codeforafrica/payload.config.ts

```diff
@@ -57,6 +58,7 @@ export default buildConfig({
     Donors,
     GuidingPrinciples,
     Impact,
+    Locations,
```

Comment:
Would a better option be to make it a global since it's a global setting?

Reply:
We have more than one office so I think a `collection` `Offices` is alright.

Global would also work if we use an array of offices or something but collection is more straight forward for content editors as it works similar to pretty much every other content on the CMS.